plugins/sigma.renderers.linkurious/canvas/sigma.canvas.nodes.def.js

```diff
@@ -157,9 +169,9 @@
       // Border:
       if (borderSize > 0) {
         context.beginPath();
-        context.fillStyle = settings('nodeBorderColor') === 'node' ?
-          (color || defaultNodeColor) :
-          settings('defaultNodeBorderColor');
+        context.fillStyle = settings('nodeBorderColor') === 'node'
+          ? borderColor
```

Comment:
I just noticed that in the definition of `color` you give precedence to `o.color` - shouldn't `node.color` have the highest precedence, since that allows per-node customisation?

```
color = node.color || o.color || defaultNodeColor,
```
instead of the current:
```
color = o.color || node.color || defaultNodeColor,
```